docs/docs/build/contracts/index.md

```diff
@@ -2,27 +2,15 @@
 
 ## Mainnets
 
-### Ethereum (L1)
+### BOB Core Contracts (L2)
 
-- SystemConfig: [0xaa0A1EfD35d6578ea6B5704dbc2c40B36A55B590](https://etherscan.io/address/0xaa0A1EfD35d6578ea6B5704dbc2c40B36A55B590)
-- SystemConfigProxy: [0xACB886b75D76d1c8d9248cFdDfA09b70C71c5393](https://etherscan.io/address/0xACB886b75D76d1c8d9248cFdDfA09b70C71c5393)
-- L2OutputOracleProxy: [0xdDa53E23f8a32640b04D7256e651C1db98dB11C1](https://etherscan.io/address/0xdDa53E23f8a32640b04D7256e651C1db98dB11C1)
-- OptimismPortal: [0x994e3B01D130944a3E67BFd3B8Fc73069b959FEc](https://etherscan.io/address/0x994e3B01D130944a3E67BFd3B8Fc73069b959FEc)
-- OptimismMintableERC20FactoryProxy: [0x5557408ab14013ce9Dbb300dE0D87D386BB09cb6](https://etherscan.io/address/0x5557408ab14013ce9Dbb300dE0D87D386BB09cb6)
-- L1StandardBridgeProxy: [0x3F6cE1b36e5120BBc59D0cFe8A5aC8b6464ac1f7](https://etherscan.io/address/0x3F6cE1b36e5120BBc59D0cFe8A5aC8b6464ac1f7)
-- L2OutputOracle: [0x59191bD38EBA4a642C9FEc308dc188731b229822](https://etherscan.io/address/0x59191bD38EBA4a642C9FEc308dc188731b229822)
-- L1CrossDomainMessengerProxy: [0xE3d981643b806FB8030CDB677D6E60892E547EdA](https://etherscan.io/address/0xE3d981643b806FB8030CDB677D6E60892E547EdA)
-- AddressManager: [0xF2dc77c697e892542cC53336178a78Bb313DFDC7](https://etherscan.io/address/0xF2dc77c697e892542cC53336178a78Bb313DFDC7)
-- L1CrossDomainMessenger: [0x237853621998a33Fa5B9B820592F4c6f4c158c12](https://etherscan.io/address/0x237853621998a33Fa5B9B820592F4c6f4c158c12)
-- ProxyAdmin: [0x0d9f416260598313Be6FDf6B010f2FbC34957Cd0](https://etherscan.io/address/0x0d9f416260598313Be6FDf6B010f2FbC34957Cd0)
-- L1ERC721BridgeProxy: [0x5fF93263D5181b2A826f8c51d54BC0da2d20D50a](https://etherscan.io/address/0x5fF93263D5181b2A826f8c51d54BC0da2d20D50a)
-- OptimismPortalProxy: [0x8AdeE124447435fE03e3CD24dF3f4cAE32E65a3E](https://etherscan.io/address/0x8AdeE124447435fE03e3CD24dF3f4cAE32E65a3E)
-- OptimismMintableERC20Factory: [0x0640c9377a4fAC00225097EACbB0295f4D5fbd76](https://etherscan.io/address/0x0640c9377a4fAC00225097EACbB0295f4D5fbd76)
-- L1StandardBridge: [0xEEAfA156A5dd3811Ee0D9F91db57A77eA53A8d31](https://etherscan.io/address/0xEEAfA156A5dd3811Ee0D9F91db57A77eA53A8d31)
-- L1ERC721Bridge: [0xA67fc05D295d2Be6bC4cBfad4f880c1c58734212](https://etherscan.io/address/0xA67fc05D295d2Be6bC4cBfad4f880c1c58734212)
+- [tBTC](/docs/build/bob-sdk/relay) Relay: [`0x9fe7ef727da3d79e0308ff43f31ea1d077ee9f41`](https://explorer.gobob.xyz/address/0x9fe7ef727da3d79e0308ff43f31ea1d077ee9f41)
+- [Gateway](/docs/build/bob-sdk/gateway): [`0xcc2c1d45A3ECeEdec364a6B9e537AE97Fa20bEa7`](https://explorer.gobob.xyz/address/0xcc2c1d45A3ECeEdec364a6B9e537AE97Fa20bEa7)
```

Comment:
Not sure about linking this, this is only one of many

Reply:
Great catch, @gregdhill. I should have checked whether it was an LP-specific address.

Which of [these contracts](https://github.com/bob-collective/bob-gateway#contracts), if any, should we add then?

Reply:
@gregdhill, please take a look when you get a chance so we can close this PR 🙏